.github/workflows/scripts/monitor-typescript.ts

```diff
@@ -3,6 +3,7 @@
 // eslint-disable-next-line
 import * as fs from 'fs';
 import * as Sentry from '@sentry/node';
+import '@sentry/tracing';
```

Comment:
Usually the linter makes you put this in a group above these other things no?

Reply:
Is it because it is under `.github/workflows/scripts/monitor-typescript.ts`? I can move it.

Reply:
yeah maybe linter isn't checking this?